jib-maven-plugin/src/test/resources/projects/complex/src/main/java/com/test/HelloWorld.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2018 Google LLC. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.test;
+
+import dependency.Greeting;
+import java.io.IOException;
+import java.lang.management.ManagementFactory;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/** Example class that uses a dependency and a resource file. */
+public class HelloWorld {
```

Comment:
If the code for this `complex` project is mostly the same as the `simple` project, it could just use the same `simple` project, but have a different POM (like `pom-complex.xml`) that can be passed into the verifier with the `--file` CLI option.